components/series/SeriesTable.vue

```diff
@@ -396,6 +398,10 @@ export default class SeriesTable extends mixins(PrefixMixin, AuthMixin) {
     await this.fetchCollectionsSeries(Number(this.nbRows))
   }
 
+  get urlPrefix() {
+    return this.$store.getters.currentUrlPrefix
+  }
```

Comment:
We would not need this function cause we could get the `urlPrefix` from the existing `PrefixMixin`

Reply:
didn't notice that's part of it, fixed now.